### PR TITLE
RF | homePage.spec | TC 01.2.1_01 Verify user can make search entered the valid text in the search field

### PR DESCRIPTION
**Pull request by gudzenkoleruska**
https://trello.com/c/OaxDVpZz/598-rf-homepagespec-tc-012101-verify-user-can-make-search-entered-the-valid-text-in-the-search-field

### CODE REVIEW

File: page_objects/homePage.js [hunk 1]
```diff
@@ -27,7 +27,6 @@ class HomePage {
     getSearchInputField: () => this.page.getByPlaceholder("Search entire store here..."),
     getWaitForAutocompleteSearchItems: () => this.page.waitForSelector("#search_autocomplete>ul>li>span:first-child"),
     getAutocompleteSearchItems: () => this.page.locator("#search_autocomplete>ul>li>span:first-child"),
-    getSearchButton: () => this.page.locator('button[title="Search"]'),
```
Comment (Sensation3000) [inline page_objects/homePage.js:30]: hi. Why did you remove this locator?

Comment (Sensation3000) [reply page_objects/homePage.js:30]: this locator is used in the other test 
image

Comment (verafes) [reply page_objects/homePage.js:30]: There is the locator getSearchButton already exists (see line 26 above). Please use unique names for your locators or re-use existing ones. Thanks

**Review by verafes — APPROVED** [review]

File: page_objects/homePage.js (hunk 1 above)
Comment (Sensation3000) [reply page_objects/homePage.js:30]: oh, i see

**Review by Sensation3000 — DISMISSED** [review]

File: page_objects/homePage.js (hunk 1 above)
Comment (gudzenkoleruska) [reply page_objects/homePage.js:29]: yes, it was the totally duplicate of already exist locator, that is why i deleted this duplicate

**Review by verafes — APPROVED** [review]

**Review by Sensation3000 — APPROVED** [review]
